### PR TITLE
Bugfix/lattice 2154 materialized entityset org update

### PR DESCRIPTION
**Pull request by you-leee**

### CODE REVIEW

File: src/test/java/com/openlattice/rehearsal/ThrowingCallAdapterFactory.kt [hunk 1]
```diff
@@ -25,41 +25,57 @@ import com.google.common.base.Charsets
...
 
-        }
+            }
 }
 
-class GeneralException(message: String):java.lang.Exception(message)
\ No newline at end of file
+class GeneralException(message: String) : java.lang.Exception(message)
+
+internal fun assertException(fqn: () -> (Any), expectedMsg: String) {
```
Comment (makosblade) [inline src/test/java/com/openlattice/rehearsal/ThrowingCallAdapterFactory.kt]: Maybe consider making this an `inline` function. I'm not sure if it's too big of a function for it but it should offset any performance penalty from passing that `()->(Any)` lambda in and it's called quite a lot

File: src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt [hunk 2]
```diff
@@ -68,57 +69,15 @@ class EdmTest : MultipleAuthenticatedUsersBase() {
         Assert.assertEquals(setOf<UUID>(), entitySetsApi.getEntitySet(linkingEs.id).linkedEntitySets)
     }
 
-    @Test
-    fun testChecksOnAddAndRemoveLinkedEntitySets() {
-        // entity set is not linking
```
Comment (makosblade) [inline src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt:73]: Did this group of tests move elsewhere?

**Review by makosblade — DISMISSED** [review]

File: src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt (hunk 2 above)
Comment (you-leee) [reply src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt:73]: Yep, it was also present in EntitySetsTest

**Review by katherinebernstein — DISMISSED** [review]

File: src/test/java/com/openlattice/rehearsal/ThrowingCallAdapterFactory.kt (hunk 1 above)
Comment (you-leee) [reply src/test/java/com/openlattice/rehearsal/ThrowingCallAdapterFactory.kt]: 👍 Cool, will do!

**Pull request merged**